install_goto.sh

```diff
@@ -8,7 +8,32 @@ touch ${HOME}/.goto/active-project
 ln -s /usr/local/opt/goto/bin/* /usr/local/bin/
 
 # add init_script to.bash_profile:
-echo "source start_goto" >> ${HOME}/.bash_profile
+
+
+if [ -f "${HOME}/.bash_profile" ]; then
+	echo 
+	echo "Next step is required to make goto work:"
+	echo
+	read -p "Add goto startup script to .bash_profile? [yes|no]: " -n 1 -r
```

Comment:
[yes|no]  should be [y|n] or [y]es | [n]o , since only expecting 1 character for the user.